### ISSUE TITLE
Disclosure pattern - Partially hidden content

### ISSUE THREAD

**Issue by gaurav5430** [w3c/aria#1313]
All the examples on https://www.w3.org/TR/wai-aria-practices-1.1/#disclosure talk about patterns which show/hide the complete dependent content.

Are the same practices also applicable for partially hidden content. Can a Read more / Read less button (and the content it controls) be considered a disclosure pattern example? 

Something like this:

<img width="824" alt="Screenshot 2020-08-19 at 4 52 52 PM" src="https://user-images.githubusercontent.com/12510548/90629163-8249a100-e23c-11ea-830d-4c1a2bdb8545.png">
<img width="827" alt="Screenshot 2020-08-19 at 4 52 59 PM" src="https://user-images.githubusercontent.com/12510548/90629171-84136480-e23c-11ea-9266-f22d8c06611c.png">

**Comment by carmacleod** [comment]
Yep, that's a disclosure. The main idea is to have aria-expanded="true"/"false" on the button.

I'm not quite sure how users will discover that the added content is _before_ the button, though.
Adding aria-controls="id-of-added-content" may help with that in some screen readers.
I don't think you would want to go the route of putting the added content after the button in the DOM and then CSS'ing it into place... that seems like a bad hack.

Hopefully others will chime in here.

**Comment by scottaohara** [comment]
yes, still a type of disclosure widget, but changing the name of the control as well as the state would be a bit awkward though.   

might this be more appropriately filed over in APG vs the ARIA spec?

**Comment by carmacleod** [comment]
> changing the name of the control as well as the state would be a bit awkward though.

Good point: "Read more, collapsed" and "Read less, expanded" would be confusing.

> might this be more appropriately filed over in APG vs the ARIA spec?

Also a good point. :)
